pkg/metrics/collector.go

```diff
@@ -591,6 +623,40 @@ func (c *ResourcesMetricsCollector) collectConfigAuditReports(ctx context.Contex
 	}
 }
 
+func (c *ResourcesMetricsCollector) collectConfigAuditInfoReports(ctx context.Context, metrics chan<- prometheus.Metric, targetNamespaces []string) {
+	reports := &v1alpha1.ConfigAuditReportList{}
+	labelValues := make([]string, len(c.configAuditInfoLabels))
+	for _, n := range targetNamespaces {
+		if err := c.List(ctx, reports, client.InNamespace(n)); err != nil {
+			c.Logger.Error(err, "failed to list configauditreports from API", "namespace", n)
+			continue
+		}
+		for _, r := range reports.Items {
+			if c.Config.MetricsConfigAuditInfo {
+				labelValues[0] = r.Namespace
+				labelValues[1] = r.Name
+				labelValues[2] = r.Labels[trivyoperator.LabelResourceKind]
+				labelValues[3] = r.Labels[trivyoperator.LabelResourceName]
+				for _, config := range r.Report.Checks {
+					labelValues[4] = config.ID
+					labelValues[5] = config.Title
+					labelValues[6] = config.Description
+					labelValues[7] = config.Category
+					labelValues[8] = strconv.FormatBool(config.Success)
+				}
+				for i, label := range c.GetReportResourceLabels() {
+					labelValues[i+5] = r.Labels[label]
+				}
+				for severity, countFn := range c.configAuditSeverities {
+					labelValues[9] = severity
+					count := countFn(r.Report.Summary)
+					metrics <- prometheus.MustNewConstMetric(c.configAuditInfoDesc, prometheus.GaugeValue, float64(count), labelValues...)
```

Comment:
metric count for `collectConfigAuditInfoReports` is always `float64(1)`  as there is 1 count only for each entry